### PR TITLE
Update README.md

### LINKED ISSUE #815: Missing information of localized build for Typescript users.
In our project we are using **Typescript** with **Vue** and need to support **Traditional Chinese** language (`zh-TW`) locale. Since Typescript requires type checking to work. Dynamically create and append script tag to load the localized version is not an option. Therefore we need to pull the repo and build by our own. Following the `README.md` doesn't work for us, with `firebaseui__zh_tw.js` and `npm__zh_tw.js`, the UI were still in **English** (`en`), when inspected the element was still look like this:

<pre>
&lt;div id="firebase-auth-container" lang="en"&gt;
</pre> 

After a few hours of trial and error we find that `esm.js` was the file we actually needed to be localized. However the `README.md` didn't mention we can instead build the localized version of `esm.js` via `yarn build build-esm-zh-tw` as well. I think this information will help both the new user of **firebaseui** and **TypeScript** a lot.

Would the repo owner mind I do a PR to update the `README.md` with steps on how I successfully build, import and use the localized version?

Below is my successful screenshot:
image
image

### PR DESCRIPTION
**Pull request by ferrywlto**
- Add missing section of localized build for ES module.
resolves #815

### CODE REVIEW

**Comment by ferrywlto** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -1660,6 +1660,77 @@ npm run build build-npm-zh-TW
...
+will be mapped to `zh-TW`, `xh-CN`, `pt-PT`. The underscore will be replaced by
+a hyphen symbol and the subsequent characters will be capitalized.
+
+```bash
+npm run build build-esm-zh-TW
+```
+
+This will create a binary `esm__zh_tw.js` in the `dist/` folder.
+
+### Tips for **TypeScript** users using localized build.
```
Comment (bojeil-google) [inline README.md:1687]: Can you please remove this section `Tips for **TypeScript** users using localized build`. It is a very unique use case. CDN is not typically used with typescript and the slated instructions seem to be very custom for your use case.

**Review by bojeil-google — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bojeil-google) [reply README.md:1687]: Another option is to keep the section but remove some of the details regarding using both CDN and npm/esm build at the same time to keep things simple and straight to the point.

Comment (ferrywlto) [reply README.md:1687]: Okay I will remove that part.

**Review by bojeil-google — APPROVED** [review]

**Comment by bojeil-google** [comment]
Thank @ferrywlto for the contribution!

**Pull request merged**